scripts/manage-db.py

```diff
@@ -84,13 +85,57 @@ def cleanup_releases_history(trans, dryrun=True):
 
     print "Total Deleted: %d" % total_deleted
 
+
+def extract_active_data(URL, loc="dump.sql"):
+    """
+    Function was added in to enhance testing in Balrog's stage environment.
+
+    :param URL: Database. eg : mysql://balrogadmin:balrogadmin@balrogdb/balrog
+    :param loc: location where sqldump file must be created
+    :return: Stores sqldump data in  the specified location, if not specified stores in current directory in file dump.sql
+            If  file already exists it will override that file and not append it.
+    """
+    url = make_url(URL)
+    user = url.username
+    password = url.password
+    host = url.host
+    database = url.database
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s rules rules_history rules_scheduled_changes \
```

Comment:
I just noticed that we added another table recently. Can you add "dockerflow" to this list as well?